app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center : float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings : int
+                 ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        station_income = 0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                station_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(station_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class * (self.clean_power - car.clean_mark)
+                 * self.average_rating / self.distance_from_city_center)
```

Comment:
Ensure that `distance_from_city_center` is not zero to avoid division by zero errors. Consider adding a check or a default value to handle this case.